env.default

```diff
@@ -1,3 +1,5 @@
 PUID=1000
 PGID=1000
 PHP_TZ=UTC
+CACHE_DRIVER=file
+LOG_VIEWER_CACHE_DRIVER=file
```

Comment:
Pretty sure these don't belong here. These are just for testing and are Docker env, not Lychee.